src/js/LoginSignupUtils.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * passwordInput의 type을 text와 password로 toggle하는 함수
+ * @param {input} passwordInput - 비밀번호 input
+ * @param {span} passwordIcon - 비밀번호 아이콘
+ */
+export const togglePasswordIcon = (passwordInput, passwordIcon) => {
+  const type = passwordInput.type == 'password' ? 'text' : 'password';
+  passwordInput.type = type;
+  passwordIcon.style.backgroundImage =
+    type ==
+      'password' ?
+      "url('/src/assets/img/btn_visibility_off.svg')" :
+      "url('/src/assets/img/btn_visibility_on.svg')";
+}
+
+/**
+ * passwordCheckInput의 type을 text와 password로 toggle하는 함수
+ * @param {input} passwordCheckInput - 비밀번호 확인 input
+ * @param {span} passwordCheckIcon - 비밀번호 확인 아이콘
+ */
+export const togglePasswordCheckIcon = (passwordCheckInput, passwordCheckIcon) => {
+  const type = passwordCheckInput.type == 'password' ? 'text' : 'password';
+  passwordCheckInput.type = type;
+  passwordCheckIcon.style.backgroundImage = 
+    type ==
+      'password' ?
+      "url('/src/assets/img/btn_visibility_off.svg')" :
+      "url('/src/assets/img/btn_visibility_on.svg')";
+}
+
+/**
+ * emailInput의 value가 비어있거나 '@'를 포함하지 않을 때 에러메시지를 보여주는 함수
+ * @param {input} emailInput - 이메일 input
+ * @param {p} emailErrorMessage - 이메일 에러 메시지
+ */
+export const showEmailErrorMessage = (emailInput, emailErrorMessage) => {
+  if (emailInput.value == '') {
+    emailErrorMessage.textContent = '이메일을 입력해주세요.';
+    emailErrorMessage.style.display = 'block';
+  } else if (!emailInput.value.includes('@')) {
```

Comment:
P2:
early return 문을 활용하셔도 동일동작할 것 같습니다.

```suggestion
  if (!emailInput.value.trim()) { // 가능하면 완전 일치 연산자를 써주시고 이렇게도 작성 가능합니다.
    emailErrorMessage.textContent = '이메일을 입력해주세요.';
    emailErrorMessage.style.display = 'block';
    return;
  }
  
  if (!emailInput.value.includes('@')) {
```